public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -184,6 +188,41 @@ class DefineMonitor extends Component {
     }
   }
 
+  async onRunHttp() {
```

Comment:
There is a lot of code in here that shouldn't be, it seems like it was just copy pasted from a different component. Please rewrite this method to reflect what you're trying to do.